### ISSUE TITLE
pyinstaller cannot find pywin32-ctypes

### ISSUE THREAD

**Issue by pe2mbs** [pyinstaller/pyinstaller#3892]
Hi, when i try to create a package with the following command i get the error below. Even that all the packages are installed (see pip3 list). I'm running 

**Python 3.6.6 (v3.6.6:4cf1f54eb7, Jun 27 2018, 02:47:15) [MSC v.1900 32 bit (Intel)] on win32**

All packages are installed via PyPi, except for PyInstaller that was install using the link "https://github.com/pyinstaller/pyinstaller/tarball/develop", the same problem is on the PyInstaller from PyPi

> pyinstaller --onefile myservice.py

```
PyInstaller cannot check for assembly dependencies.
Please install pywin32-ctypes.

pip install pywin32-ctypes
```

> pip3 list
Package                Version
---------------------- ------------------
alembic                0.8.4
altgraph               0.16.1
apispec                0.39.0
asn1crypto             0.24.0
bcrypt                 3.1.4
blinker                1.4
cached-property        1.5.1
certifi                2018.10.15
cffi                   1.11.5
chardet                3.0.4
Click                  7.0
colorama               0.4.1
cryptography           2.3.1
cx-Oracle              7.0.0
defusedxml             0.5.0
dnspython              1.15.0
exchangelib            1.12.0
Flask                  1.0.2
flask-apispec          0.7.0
Flask-Bcrypt           0.7.1
Flask-Builder          0.9
Flask-Caching          1.4.0
Flask-CLI              0.4.0
Flask-Cors             3.0.6
Flask-JWT              0.3.2
Flask-JWT-Extended     3.13.1
Flask-Mail             0.9.1
flask-marshmallow      0.9.0
Flask-Migrate          2.2.1
Flask-SQLAlchemy       2.3.2
future                 0.17.1
gunicorn               19.9.0
idna                   2.7
isodate                0.6.0
itsdangerous           1.1.0
Jinja2                 2.10
lxml                   4.2.5
macholib               1.11
Mako                   1.0.7
MarkupSafe             1.0
marshmallow            2.15.6
marshmallow-sqlalchemy 0.14.1
ntlm-auth              1.2.0
numpy                  1.15.3
pandas                 0.23.4
pefile                 2018.8.8
pip                    18.1
psutil                 5.4.8
psycopg2               2.7.5
pycparser              2.19
Pygments               2.2.0
PyInstaller            3.5.dev0+e29c04727
PyJWT                  1.6.4
pypiwin32              223
pyreadline             2.1
python-dateutil        2.7.5
python-dotenv          0.9.1
python-editor          1.0.3
pytz                   2018.7
pywin32                224
pywin32-ctypes         0.2.0
PyYAML                 3.13
raven                  6.9.0
requests               2.20.0
requests-ntlm          1.1.0
setuptools             39.0.1
six                    1.11.0
SQLAlchemy             1.2.12
tzlocal                1.5.1
unicode-slugify        0.1.3
Unidecode              1.0.22
urllib3                1.24
waitress               1.1.0
webargs                4.1.1
Werkzeug               0.14.1

**Comment by pe2mbs** [comment]
Ok, fixed the issue, deleted brute force the 'pywin32_ctypes-0.2.0.dist-info' and 'win32ctypes' folders from 'site-packages'.
pip3 install pywin32-ctypes

**Comment by BrendanSimon** [comment]
The solution of brute force delete of the files did not work for me.

I've tried so many combinations of PyInstaller (release and develop), pywin32, pywin32-ctypes, pypiwin32.

Nothing seems to get rid of this problem.

I have multiple installs of Python.  Python36-32, Python36, Python37-2 and Python37.  None of the pythons are in my PATH.  I have uninstalled PyInstaller from all installations, and reinstalled PyInstaller only for the install I'm interested in (Python36-32).

How, How, How can I get rid of this road-blocking issue ?
